### PR TITLE
GIVCAMP-245 | Feature Masonry media grid

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Just initial commit for the feature masonry media grid for the featured story
- They're still changing the design, so I'm going to stop working on the details until they finalize
- Merging this in since the page is published on Storyblok and the component has been added.

# Review By (Date)
- Retro

# Review Tasks

## Setup tasks and/or behavior to test

1. https://deploy-preview-149--giving-campaign.netlify.app/stories/education-for-a-life-of-purpose
2. Scroll down to see the 5 item masonry grid thingy

# Associated Issues and/or People
- JIRA ticket(s) - GIVCAMP-245

### CODE REVIEW

**Comment by yvonnetangsu** [comment]
@sherakama This is initial commit only since the design still is changing quite a bit. I'm going to merge this in and focus on something else first 😄

File: components/FeaturedStories/FeatureMasonry.styles.ts
Comment (sherakama) [inline components/FeaturedStories/FeatureMasonry.styles.ts:1]: So much empty

Comment (yvonnetangsu) [reply components/FeaturedStories/FeatureMasonry.styles.ts:1]: 😆 File is there, classnames will get change around so I'll move them to the file at the end. I thought it's ready to be build, but the team is still making a lot of changes.

**Pull request merged**